packages/client/CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.4.1 - 2022-04-14
+
+- Allow `forkchoiceUpdated` method to properly work on a reorg if a previous block was sent as `headBlockHash`, PR [#1820](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1820)
+- Fix to have `newPayload` properly handle already existing payload, PR [#1824](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1824)
+- client/engine: add ability to add blocks to blockchain without setting the head (faster engine calls!), PR [#1827](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1827)
+- chain: safer closing to not cause db corruption during shutdown while handling engine requests, PR [#1827](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1827)
+- CLConnectionManager: starts on first `updateStatus` in case `preMerge` isn't reached before merge (Goerli shadow fork scenario), PR [#1827](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1827)
```

Comment:
Let's make sure to update `preMerge` to whatever we end up calling the forkhash before the Merge (`mergeForkBlock` or `mergeForkTransitionId` or whatever)

Reply:
updated, thx!